src/Tasks/ResourceHandling/MSBuildResXReader.cs

```diff
@@ -192,7 +192,7 @@ private static void ParseData(string resxFilename, bool pathsRelativeToBasePath,
                         resources.Add(new BinaryFormatterByteArrayResource(name, binaryFormatterBytes, resxFilename));
                         return;
                     default:
-                        throw new NotSupportedException($"Resource \"{name}\" in \"{resxFilename}\"uses MIME type \"{mimetype}\", which is not supported by .NET Core MSBuild.");
+                        throw new NotSupportedException(ResourceUtilities.FormatResourceStringIgnoreCodeAndKeyword("GenerateResource.MimeTypeNotSupportedOnCore", name, resxFilename, mimetype));
```

Comment:
I think if we're going to localize it we should instead log an error here (so this could depend on your other PR). I suspect I did it this way originally because that wasn't available here and then I didn't fix it.

Reply:
I can draft it for now, then.